### DISCUSSION TITLE
Combined analysis of metagenome and metatranscriptome

### DISCUSSION THREAD

**Discussion by jilldv** [metagenome-atlas/atlas#659] [Q&A] [answered]
Hello,

I have two datasets from partly the same samples. The metagenomics dataset is from samples from two different sites, all samples were taken during summer. The metatranscriptomics dataset is from samples from one of the sites only, but taken in different seasons. I am running Atlas on the metagenomic dataset now, with both genomes and genecatalog workflows. But I am wondering how to best analyze both datasets together or how to integrate the results from both datasets. The aim is to compare the communities and functions between both sites. And for the site where we have both datasets, compare the functions betweens seasons.

After looking around a bit in the discussions here and the Atlas documentation I understood it is possible to run the genecatalog workflow on metatranscriptome data, if I understood correctly? 
On the other hand, I also noticed some people map their metatranscriptome reads to the corresponding metagenome. Since both my datasets have somewhat different samples I am not sure if this would be an option. Also on this approach, I was wondering if this is possible with Atlas (since my metagenome dataset will already be analyzed with Atlas)?

Any advice would be very much appreciated.

Best regards

**Comment by SilasK** [comment, marked as answer]
Thank you for bringing this up. Much could be done with metagenome + metatranscriptome data, and atlas only has a fraction of the functionalities. I might implement some more soon.

What you can do today with your data:

  1. Metagenome
     - Run atlas genecatalog and genomes on your meta**genome** dataset. 
     - Activate all annotations that you want in config file. 
     - (Try out `vamb` binner)
     - Check mapping rates. 
     - Compare the two sites (See the tutorial which I will update soon)
   2. Run atlas qc on Meta**transcriptomes** in two new projects (rerun atlas init). Check that sample names makes sense. sometime atlas renames them. 
   3. Map Metatranscriptome -> genomes
        1.  Specify `genome_dir=path/to/working_dir_metagenome` in config file of the meta**transcriptome** project.
        2. Run `atlas run None quantify_genomes` on metatranscriptome


   4. Map Metatranscriptome -> genecatalog
        1.  Create a `Genecatalog` folder in the meta**transcriptome** projects.
        2. Copy or symlink the `Genecatalog/gene_catalog.fna` from the metagenome to the metatranscriptome projects in th e`Genecatalog`subfolder. 
        3. Run `atlas run None "Genecatalog/counts/median_coverage.h5" ` on metatranscriptome projects. 


Always do a dry run before running atlas to see that it only executes the steps intended. 
Could you check the mapping rates? 

Depending on how separate your research questions are. You could create MAGs and Gene catalog only for the site that you want to compare to the metatranscriptome. 

I hope this helps. Otherwise ask.

**Comment by SilasK** [comment]
What soon could be done is assembling metatranscriptome data to a gene catalog. This could give you a more comprehensive genecatalog.

**Comment by jilldv** [comment]
Thank you for your reply, this is very helpful. It looks like there is much more to do than I was thinking about. 
I will also take a look at the gene catalog from metatranscriptome data, thank you for the suggestions!

**Comment by jilldv** [comment]
I have a question about mapping the metatranscriptome to the genomes. Where in the config file do I specify `genome_dir=path/to/working_dir_metagenome`? 
If I specify it under `genecatalog: source: path/to/working_dir_metagenome`
and I do a dry run it seems atlas also runs metabat and maxbin, which doesn't seem right? Binning the metatranscriptome would not make much sense I would think?

**Comment by SilasK** [comment]
put 
```
genome_dir: "path/to/working_dir_metagenome/genomes/genomes"
```
anywhere in the config file but at the base level. Very left, so not under genecatalog.

**Reply by jilldv** [reply]
Thank you very much, now it runs the correct steps
